media/ivf.ksy

```diff
@@ -0,0 +1,67 @@
+meta:
+  id: ivf
+  file-extension: ivf
+  endian: le
+  title: VP8 raw file format
```

Comment:
Please remove word "format" from the title — just "VP8 raw file" will work, as in certain contexts this "title" would be used as `$title format`.